projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -408,15 +416,35 @@ export class IgxTimePickerComponent implements
      *<igx-toast #toast message="The time picker has been opened!"></igx-toast>
      * ```
      */
+    @DeprecateProperty(`'onOpen' @Ouput property is deprecated\nUse 'onOpened' instead.`)
```

Comment:
Typo `@Ouput`. Replace `\n` with `.` so there won't be a new line in the output.